spec/models/claims/claim_spec.rb

```diff
@@ -110,4 +110,41 @@
       end
     end
   end
+
+  describe "#ready_to_be_checked?" do
+    it "returns true if the claim is ready to be checked" do
+      provider = create(:provider)
+      mentor = create(:claims_mentor)
```

Comment:
Do these need to be explicit? I think we can rely on the `mentor_training` factory to create these.

We can pass the `claim.provider` through to `create(:mentor_training, provider: claim.provider)`.